assets/js/dpi.js

```diff
@@ -70,6 +71,7 @@ $u.xhr({
 		tbody.appendChild(fragment);
 	}
 });
+*/
```

Comment:
No need to comment out unneeded JS, just delete it. That's why we have version control: If we need this code at some point, we can go back in history and get it. :)
Also, I'm pretty sure there's more code that could be removed. E.g. the code that did the searching or the `deviceRow()` function.